client/src/main/java/org/eclipse/hono/client/impl/ProtocolAdapterCommandConsumerFactoryImpl.java

```diff
@@ -81,7 +80,7 @@
     private final AtomicBoolean recreatingConsumers = new AtomicBoolean(false);
     private final AtomicBoolean tryAgainRecreatingConsumers = new AtomicBoolean(false);
 
-    private BasicDeviceConnectionClientFactory deviceConnectionClientFactory;
+    private CommandHandlingAdapterInfoAccess commandHandlingAdapterInfoAccessor;
```

Comment:
In the comment a few lines above:
````
     * Will be used to represent the protocol adapter instance that this factory instance is used in, when registering
     * command handlers with the Device Connection service.
     */
    private final String adapterInstanceId;
````
"Device Connection service" could be replaced with "CommandHandlingAdapterInfoAccess service".

Reply:
I have pushed an update ...